infra/testApi.mjs

```diff
@@ -17,8 +17,15 @@ assert.equal(improveResponse.status, 200)
 // assert.equal(improveHTML.length, improveHTMLexpectedCount, `HTML response character count should be ${improveHTMLexpectedCount} unless HTML changes.`)
 
 // improve-start endpoint
-const startURL = `http://127.0.0.1:8000/improve-start?formula=${encodeURIComponent(FPCoreFormula2)}`
-const startResponse = await fetch(startURL, { method: 'POST' })
+const startURL = `http://127.0.0.1:8000/improve-start`
+const URIencodedBody = "formula=" + encodeURIComponent(FPCoreFormula)
```

Comment:
This is fine, there is a `FormData` API in the browser that makes this easier but it looks like it's not available in Node.

Reply:
That's good to know. Thank you.

Reply:
Anything you might suggest to make this better? I plan to wrap this up in a helper function like you suggested in another PR.